### PR TITLE
add angular-protractor to typings

### PR DESCRIPTION
**Pull request by michaelchiche**

### CODE REVIEW

**Comment by Foxandxss** [comment]
Thanks, I forgot about that, will test.

File: typings.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   },
   "globalDependencies": {
     "es6-shim": "registry:dt/es6-shim#0.31.2+20160317120654",
+    "angular-protractor": "registry:dt/angular-protractor#1.5.0+20160425143459",
```
Comment (rantrix) [inline typings.json]: maybe add this in alphabetical order?

**Comment by Foxandxss** [comment]
I feel like this doesn't do much unless we move the tests to typescript.

**Comment by mattdistefano** [comment]
@Foxandxss If you're using an editor that supports it, you can do a triple-slash ref within the .js files and still get some degree of code completion and such from the typings.

**Comment by Foxandxss** [comment]
I'm gonna update to protractor 4 soon. That comes with typings

**Pull request closed without merging**